src/keys_handler.rs

```diff
@@ -232,10 +232,58 @@ pub async fn u_or_v_key(
     HttpResponse::Ok().await
 }
 
+#[derive(Serialize, Deserialize)]
+struct KeylimePubkey {
+    pubkey: String,
+}
+
+#[derive(Serialize, Deserialize)]
+struct JsonPubkeyWrapper {
+    code: u32,
+    status: String,
+    results: KeylimePubkey,
+}
+
+impl JsonPubkeyWrapper {
+    fn new(pubkey: String) -> Self {
+        JsonPubkeyWrapper {
+            code: 200,
+            status: String::from("Success"),
+            results: KeylimePubkey { pubkey: pubkey },
```

Comment:
```suggestion
            results: KeylimePubkey { pubkey },
```
This will help clear the `clippy` CI error.

Reply:
OK, done!